learning/courses/quantum-business-foundations/quantum-technology.ipynb

```diff
@@ -0,0 +1,257 @@
+    "\n",
+    "### Quantum-centric supercomputers\n",
+    "\n",
+    "Going beyond single chip processors is the key to solving scale. This year we plan to introduce classical parallelized quantum computing with multiple Heron processors connected by a single control system. In 2024, we will debut Crossbill, the first single processor made from multiple chips. These are the first steps to ushering in a new era of scaling providing a clear path to 100,000 qubits and beyond with the quantum-centric supercomputer, a modular computing architecture which enables scaling, combining quantum communication and computation to increase computational capacity, while employing hybrid cloud middleware to seamlessly integrate quantum and classical workflows.\n",
+    "\n",
+    "Solving the world’s most complex problems will require a combination of classical and quantum resources. Furthermore, it will depend on the continued collaboration between industry and academia.\n",
+    "\n",
+    "## Key takeaways\n",
+    "\n",
+    "You can keep these key takeaways in mind:\n",
+    "\n",
+    "* Today’s quantum computers are not fault tolerant.\n",
+    "* Quantum Volume is a holistic measure of how good a quantum computer is. The higher the Quantum Volume, the better. Talking only about qubit count is misleading.\n",
+    "* To measure the performance of quantum computers, there are three key metrics: scale, quality, and speed.\n",
+    "* A joint IBM Quantum-UC Berkeley experiment presented evidence of IBM quantum computers delivering reliable, accurate results for challenging simulation problems at a scale of 127 qubits.\n",
+    "* Quantum-centric supercomputing means treating quantum as one piece of a broader HPC paradigm with classical and quantum working as one computational unit.\n",
+    "* The IBM Quantum Network website is a key resource that you can use to stay current on IBM Quantum technology and research.\n",
+    "* IBM Quantum has the largest fleet of quantum computers available over the cloud.\n",
+    "* IBM Quantum cloud tools allow you to accelerate your individual research by running quantum applications on the cloud."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da1985f7-155c-4d5d-bacc-c145cda087ea",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "|         |       |\n",
+    "| ----------------- | -- |\n",
+    "| [< Prev](/learning/courses/quantum-business-foundations/quantum-computing-fundamentals) | [Next >](/learning/courses/quantum-business-foundations/business-impacts) |"
+   ]
```

Comment:
I would remove these previous/next links in every file. We are trying to add this through a nicer mechanism.

Reply:
I added it because of intro section mentioned "this course will have prev and next". I deleted that part and will remove all of prev and next